components/centraldashboard/public/components/main-page.js

```diff
@@ -27,88 +28,130 @@ import '@polymer/neon-animation/animations/fade-out-animation.js';
 import css from './main-page.css';
 import template from './main-page.pug';
 
+import './dashboard-view.js';
+import './activity-view.js';
+
 /**
  * Entry point for application UI.
  */
 export class MainPage extends PolymerElement {
     static get template() {
-        return html([`<style>${css.toString()}</style> ${template()}`]);
+        return html([`
+        <style is="custom-style"
+            include="iron-flex iron-flex-alignment iron-positioning">
+        <style>${css.toString()}</style> ${template()}
+        `]);
     }
 
     static get properties() {
         return {
-            links: {
+            page: String,
+            routeData: Object,
+            subRouteData: Object,
+            iframeRoute: Object,
+            menuLinks: {
                 type: Array,
                 value: [
-                    {text: 'Home', defaultPage: true, hasDivider: true},
                     {
-                        link: 'https://www.kubeflow.org/docs/about/kubeflow/',
+                        iframeUrl: 'https://www.kubeflow.org/docs/about/kubeflow/',
```

Comment:
Could we go with `href`? Not all are guaranteed to open in the iframe (ie. this one!)

Reply:
I introduced `href` to be the attribute that is used when setting the anchor tag's href property on our UI to indicate the local "route". I changed link to `iframeUrl` since it was being assigned to the component's `iframeUrl` property. Other than the **Home** link which I moved out of the `menuLinks` list, all of the other ones opened in the iframe view in the existing code.

Reply:
In your new PR please also move out the docs page, thanks!